src/org/parosproxy/paros/extension/option/OptionsParamCertificate.java

```diff
@@ -46,12 +52,15 @@
     private static final String CERTIFICATE_BASE_KEY = "certificate";
 
     private static final String USE_CLIENT_CERT = CERTIFICATE_BASE_KEY + ".use";
+    private static final String PERSIST_CLIENT_CERT = CERTIFICATE_BASE_KEY + ".persist";
     private static final String CLIENT_CERT_LOCATION = CERTIFICATE_BASE_KEY + ".clientCertLocation";
+    private static final String CLIENT_CERT_PASSWORD = CERTIFICATE_BASE_KEY + ".clientCertPassword";

```

Comment:
It might be better to be more explicit and indicate which cert is being configured (`clientPkcs12CertLocation`, `clientPkcs12CertPassword`?), for if/when we happen to allow to specify PKCS#11 cert as well.

Reply:
From my understanding, PKCS#11 is an interface rather than a certificate+key combination. The GUI also request a driver and pin, rather than the certificates directly.

Reply:
Yes, they are initialised differently but both provide certificates, IMO it's better if the keys are explicit (e.g. `certificate.pkcs12.path`, `certificate.pkcs12.password`, `certificate.pkcs12.index`, `certificate.pkcs11.driver`, `certificate.pkcs11.pin`, `certificate.pkcs11.index`...) than not. Also, it's easier for maintenance if they are under different elements.